packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/markdownFeatures.ts

```diff
@@ -54,6 +54,8 @@ function cacheGetRangeForMarkdownOperation(
                 return false;
             }
 
+            //if the text is pasted, it might create a inner element inside the text element,
```

Comment:
Is it possible to have two or more layers of DIV?

Reply:
if we have to or more layers, the markdown plugin will not work. It only work for inline elements.